.devcontainer/Dockerfile

```diff
@@ -0,0 +1,16 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+# To fully customize the contents of this image, use the following Dockerfile instead:
+# https://github.com/microsoft/vscode-dev-containers/tree/v0.106.0/containers/typescript-node-12/.devcontainer/Dockerfile
+FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:0-12
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update \
+   && apt-get -y install --no-install-recommends git python-minimal build-essential \
+   && apt-get autoremove -y \
+   && apt-get clean -y \
```

Comment:
```suggestion
```

Not needed

Reply:
Why not?

Reply:
please see here: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get

Reply:
It doesn't hurt though. But I removed it nevertheless